modules/aws/jumpbox/jumpbox.userdata

```diff
@@ -29,7 +29,7 @@ write_files:
       #!/bin/bash
       curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
       install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
-      ${docker_login}
+      aws ecr get-login-password --region ${region} | docker login --username AWS --password-stdin ${registry}
```

Comment:
This is moved from the `.tf` file to make user_data idempotent.

Reply:
@nacx i feel like the https://github.com/tetrateio/tetrate-service-bridge-sandbox/blob/33e90ef57aa1135fd78fd97696739f98f223fb99/modules/aws/jumpbox/jumpbox.userdata#L45 is the issue, as it does include internal token that is being dynamically generated each run, but docker_login is static in the nature when file is rendered...

I am checking it now...

Reply:
I just checked indeed, it is the root cause, so we don't need to strip off docker_login. Technically keeping it there, minimizes the drift between the environments, do you mind rolling back to retain docker_login?

As we discussed on slack, perhaps the easiest way it is to have `aws_eip` elastic ip assigned to jumpbox to retain the SSH target.

Reply:
Ah, right, actually I had this change before, but then got to the same conclusion and reported https://github.com/tetrateio/tetrate-service-bridge-sandbox/issues/241. I'll revert this lines back.

Reply:
Done